services/frontend/www-app/src/utils/models.ts

```diff
@@ -66,8 +66,12 @@ const addressKeys = [
 ];
 
 export async function mapFeatureToPlace(
-  feature: GeoJSON.Feature,
+  feature?: GeoJSON.Feature,
```

Comment:
It doesn't make sense to call this function with nil, I think this should be reverted and the optional check moved up a level.